DMTN-107.tex

```diff
@@ -293,7 +303,7 @@ \subsubsection{Prioritize Single-Filter Template-Building During Commissioning}\
 \clearpage
 \subsection{LOY1 Data Templates}\label{ssec:potsol_cont}
 
-{\bf Build templates during on a $\sim$monthly basis as data is obtained in LOY1. Generate alerts if and when a template image exists for a field.} This option is an extension of building templates with commissioning data (\S~\ref{ssec:potsol_comm}).
+{\bf Build rolling templates on a $\sim$monthly basis as data is obtained in LOY1. Generate alerts if and when a template image exists for a field.} This option is an extension of building templates with commissioning data (\S~\ref{ssec:potsol_comm}).
```

Comment:
again I don't like the term "rolling templates"

Reply:
resolved by using "prefatory"